addons/binding/org.openhab.binding.oceanic/README.md

```diff
@@ -0,0 +1,77 @@
+# Oceanic Binding
+
+This binding integrates the Oceanic water softener and management system
+
+## Supported Things
+
+Softener
```

Comment:
Could you at least write a full sentence here? Maybe also mention models that are available and which should work, etc.